exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/metric_exporter/__init__.py

```diff
@@ -180,8 +185,8 @@ def _translate_data(
                             )
                             pb2_metric.sum.data_points.append(pt)
                     else:
-                        _logger.warn(
-                            "unsupported datapoint type %s", metric.point
+                        _logger.warning(
+                            "unsupported datapoint type %s", metric.data
```

Comment:
`metric.data` is entire class not point type right?

Reply:
I would think it would be something like `metric.data.__class__.__name__`? Also maybe change the message to `unsupported data type` as we are describing the metric data and not the data point specifically?